src/Medidata.ZipkinTracer.Core/Properties/AssemblyInfo.cs

```diff
@@ -37,7 +37,7 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.3.0")]
-[assembly: AssemblyFileVersion("1.3.0")]
-[assembly: AssemblyInformationalVersion("1.3.0")]
+[assembly: AssemblyVersion("1.4.0")]
+[assembly: AssemblyFileVersion("1.4.0")]
+[assembly: AssemblyInformationalVersion("1.4.0")]
```

Comment:
You may want to bump the version to 2.0.0 as this PR will change some method signature (for example `StartClientTrace()`) - which will make it incompatible with the previous (major) version. Also the TraceProvider changed to an internal class instead of an external dependency so this also can cause some incompatibilities.